### ISSUE TITLE
Hidden class in theme.less

### ISSUE THREAD

**Issue by carolineboeykens** [UniversalViewer/universalviewer#886]
**UV version:**

```
 universalviewer@4.0.9
```

**Current behavior:**

There is a .hidden class in the theme.less that is overriding our media queries classes:

<img width="637" alt="Screenshot 2022-10-12 at 13 56 58" src="https://user-images.githubusercontent.com/42932673/195338014-2e8ff028-2ead-48c2-8349-d32a6a2d6572.png">

![Screenshot 2022-10-12 at 14 01 33](https://user-images.githubusercontent.com/42932673/195338026-67a22f01-6284-416e-b44e-817230fa9a3d.png)

**Expected behavior:**

Can you add a prefix to this class or make it more specific like .uv .hidden?

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.

**Comment by carolineboeykens** [comment]
Yes please 😉

**Comment by edsilv** [comment]
Yep, this is something I've run into too. Should definitely be in the next community sprint.
